controllers/grid/users/userSelect/UserSelectGridHandler.inc.php

```diff
@@ -0,0 +1,200 @@
+		$this->addColumn(
+			new GridColumn(
+				'select',
+				'',
+				null,
+				'controllers/grid/users/userSelect/userSelectRadioButton.tpl',
+				$cellProvider,
+				array('width' => 5)
+			)
+		);
+		$this->addColumn(
+			new GridColumn(
+				'name',
+				'author.users.contributor.name',
+				null,
+				null,
+				$cellProvider,
+				array('alignment' => COLUMN_ALIGNMENT_LEFT,
+					'width' => 30
+				)
+			)
+		);
+	}
+
+
+	//
+	// Overridden methods from GridHandler
+	//
+	/**
+	 * @copydoc GridHandler::initFeatures()
+	 */
+	function initFeatures($request, $args) {
+		import('lib.pkp.classes.controllers.grid.feature.InfiniteScrollingFeature');
+		import('lib.pkp.classes.controllers.grid.feature.CollapsibleGridFeature');
+		return array(new InfiniteScrollingFeature('infiniteScrolling', $this->getItemsNumber()), new CollapsibleGridFeature());
+	}
+
+	/**
+	 * @copydoc GridHandler::loadData()
+	 */
+	protected function loadData($request, $filter) {
+		list($name) = $this->getFilterValues($filter);
+		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+		$stageId = $this->getAuthorizedContextObject(ASSOC_TYPE_WORKFLOW_STAGE);
+		$rangeInfo = $this->getGridRangeInfo($request, $this->getId());
+		$userStageAssignmentDao = DAORegistry::getDAO('UserStageAssignmentDAO'); /* @var $userStageAssignmentDao UserStageAssignmentDAO */
+		return $userStageAssignmentDao->filterUsersNotAssignedToStageInUserGroup($submission->getId(), $stageId, $this->_userGroupId, $name, $rangeInfo);
+	}
+
+	/**
+	 * @copydoc GridHandler::renderFilter()
+	 */
+	function renderFilter($request, $filterData = array()) {
+		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+		$stageId = $this->getAuthorizedContextObject(ASSOC_TYPE_WORKFLOW_STAGE);
+		$allFilterData = array_merge(
+			$filterData,
+			array(
+				'gridId' => $this->getId(),
+				'submissionId' => $submission->getId(),
+				'stageId' => $stageId,
+				'userGroupId' => $this->_userGroupId,
+			));
+		return parent::renderFilter($request, $allFilterData);
+	}
+
+	/**
+	 * @copydoc GridHandler::getFilterSelectionData()
+	 */
+	function getFilterSelectionData($request) {
+		$name = (string) $request->getUserVar('name');
+		return array(
+			'name' => $name,
+		);
+	}
+
+	/**
+	 * @copydoc GridHandler::getFilterForm()
+	 */
+	protected function getFilterForm() {
+		return 'controllers/grid/users/userSelect/searchUserFilter.tpl';
+	}
+
+	/**
+	 * @copydoc GridHandler::getRequestArgs()
+	 */
+
+	function getRequestArgs() {
+		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+		$stageId = $this->getAuthorizedContextObject(ASSOC_TYPE_WORKFLOW_STAGE);
+		return array(
+			'submissionId' => $submission->getId(),
+			'stageId' => $stageId,
+			'userGroupId' => $this->_userGroupId,
+		);
+	}
+
+
+	/**
+	 * Determine whether a filter form should be collapsible.
+	 * @return boolean
+	 */
+	protected function isFilterFormCollapsible() {
+		return false;
+	}
```

Comment:
The user select grid is already big and chunky. Let's collapse the filter on load, since most of the time they'll be looking for section editor or assistant.